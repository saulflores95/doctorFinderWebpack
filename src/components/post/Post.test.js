import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Post from './Post'

test('Post component should render as expected', () => {
  const component = shallow(<Post title='My Post'/>)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})

test('Post component should render as expected', () => {
  const component = shallow(<Post title='My Post' titleLink/>)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
