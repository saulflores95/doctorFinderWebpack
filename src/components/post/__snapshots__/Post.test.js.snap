exports[`test Post component should render as expected 1`] = `
<div>
  <h2
    className="title">
    My Post
  </h2>
  <p
    className="content" />
</div>
`;

exports[`test Post component should render as expected 2`] = `
<div>
  <h2
    className="title">
    <Link
      className="link"
      replace={false}
      to="/post/undefined">
      My Post
    </Link>
  </h2>
  <p
    className="content" />
</div>
`;
